### DISCUSSION TITLE
Binding temperature sensor to a device?

### DISCUSSION THREAD

**Discussion by sjorge** [Koenkk/zigbee2mqtt#13360] [General]
I recently got a Ubisys H1 TRV and from the docs it seems you can bind them and other sensors together.
<img width="443" alt="Screen Shot 2022-08-01 at 18 03 19" src="https://user-images.githubusercontent.com/379665/182191492-799275bf-a88d-4c68-a93e-90822fb5d0ef.png">

For what I can tell I should be able to bind say a temperature sensor that has msTemperatureMeasurement available as input to the output msTemperatureMeasurement of this device, although it doesn't seem possible.

Has anyone else ever tried this?

@Koenkk was there a white list somewhere in z2m for what can be bound to what? I vaguely remember something about that but can't find it atm.

**Comment by Koenkk** [comment]
> For what I can tell I should be able to bind say a temperature sensor that has msTemperatureMeasurement available as input to the output msTemperatureMeasurement of this device, although it doesn't seem possible.

For this to work, I would expect the the H1 has an input cluster `msTemperatureMeasurement` but this doesn't seem to be the case.

**Reply by sjorge** [reply]
Perhaps they are expecting a hvacThermostat -> hvacThermostat, they do mention binding a central wall thermostat to the TRV or a TRV -> multiple TRV and have one be the master.

**Comment by Nezz** [comment]
Hi @sjorge! Did you find a solution to this?

**Reply by sjorge** [reply]
I have not.
